Neuron/Sections/Transaction/TransactionSwitchTokenViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  SendTransactionSwitchTokenViewController.swift
+//  Neuron
+//
+//  Created by 晨风 on 2018/11/21.
+//  Copyright © 2018 Cryptape. All rights reserved.
+//
+
+import UIKit
+
+protocol TransactionSwitchTokenViewControllerDelegate: NSObjectProtocol {
+    func switchToken(switchToken: TransactionSwitchTokenViewController, didSwitchToToken token: TokenModel)
+}
+
+class TransactionSwitchTokenViewController: UIViewController {
+    @IBOutlet weak var backgroundView: UIView!
+    @IBOutlet weak var contentView: UIView!
+    @IBOutlet weak var tableView: UITableView!
+    private var tokens = [TokenModel]()
+    var currentToken: TokenModel!
+    weak var delegate: TransactionSwitchTokenViewControllerDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tokens += WalletRealmTool.getCurrentAppModel().nativeTokenList
+        tokens += WalletRealmTool.getCurrentAppModel().currentWallet!.selectTokenList
```

Comment:
I feel we should add a new function or var to `AppModel` that returns `nativeTokenList` and `selectTokenList`.